Remove the trailing semicolon so that Derby can use this untouched

etheriau opened the pull request:
There are only two places in the DBXxxFormats which use a trailing semicolor and Derby is freaking out about them. This change removes them from the query entirely and it works fine on both Derby and MySQL.

Commit:
Remove the trailing semicolon so that Derby can use this output format untouched.

etheriault committed:
Compilation fix for newer Hadoop versions (?)

etheriault committed:
Marking the members protected so that subclasses can access them.  This
allows, for example, the ability to tweak the TableDesc after creation
if you know what you are doing.

etheriault committed:
Expose the JobConf so that users can access the configuration if
required.

etheriault committed:
Adding a gradle build file for easier integration.

etheriault committed:
Made a copy of the com.etsy.cascading.tap.local.LocalTap but modified so that the JDBCTap
can be ran locally.  To use:

      Tap myTap = new LocalJDBCTap( jdbcTap.getScheme(), SinkMode.UPDATE, jdbcTap, "Description" );

And enjoy!

etheriault committed:
Prevent an NPE when the writer has not been opened due to an earlier SQL failure.

etheriault committed:
If the inbound tuple stream is smaller than the header row, fill in the rest with nulls.

etheriault committed:
There are some cases where the underlying tuple may produce more results than we are interested in.  For those cases, use the min of what we have and what we need.

etheriault committed:
Rename the lfs variable to tap

etheriault committed:
Remove unused import.

etheriault committed:
Remove config that is not relevant

etheriault committed:
Adding LocalMemorySourceTap and appropriate configuration.

etheriault committed:
Adjust the configuration to be consistent with what is expected

Commit:
Support for the Eclipse plugin. Adding the ignore path appropriately.

Commit:
Removing unused imports

Commit:
Ignoring the /bin directory

Commit:
Making eclipse happy: adding some serialVersionIds, removing unused variables, avoiding deprecated methods, etc.

etheriault committed:
Support for sending the setSinkField/getSinkField/setSourceField/getSourceField to the wrapping scheme.

etheriault committed:
Explicitly closing all resources, and initiating a rollback when required. Exceptions in derby resulted in an exception and warning if the other items were not closed.

etheriault committed:
Marking the getConnection as protected so that derived classes can re-use it.

etheriault committed:
Adding setters for the DBOutputFormat and DBInputFormat, allowing those to be tweaked from the caller after the fact.

Commit:
Adding an uploadArchives target to allow it to be uploaded to conjars.  See http://conjars.org/com.twitter.eyt/maple

azymnis commented:
Hey looks like there are a number of changes here, other than the trailing semicolon. Are you sure you are working off the right branch (scalding/develop)?

etheriault committed:
Merge remote-tracking branch 'upstream/master' into develop

Conflicts:
	src/jvm/com/twitter/maple/jdbc/db/DBOutputFormat.java

etheriault committed:
Forward the retrieveSourceFields and presentSourceFields to the wrapped
JDBCTap.

etheriault committed:
Fix a potential NPE where if the fieldNames were not to be initialized
(a configuration issue), use SELECT * instead.  This works for us as the
particular case in question does not use this class.

etheriault committed:
When updating the sink fields, also set the source fields.

etheriault committed:
Instead of all of the previous LocalXxxTaps, just making one
LocalTapAdapter. I've not cut everyone over to it yet, but I have some
code using this one fine.

Commit:
Moving a couple things to log.debug

Commit:
Fix warning on mavenRepo for gradle 1.8 and above

Commit:
Fix warnings

Commit:
Allowing for customization of the repo URL

etheriault committed:
If the connection is null, don't try to rollback

etheriault committed:
Merge branch 'develop' of github.com:etheriau/maple into develop

etheriault committed:
Exposing some members as protected to allow better subclass resolution

etheriault committed:
Eclipse warning cleanup

etheriault committed:
Allowing more overriding of some generic properties

etheriault committed:
The DBInputFormat was being called twice and leaking a connection on the
second time.  This resolves that issue.  This also adds many more
close's to ensure that we're closing them everywhere

Commit:
bump the ver

johnynek commented:
We're moving this code here: https://github.com/twitter/scalding/pull/779

but we are deprecating JDBC and recommending that people move to cascading-jdbc.
